substra/cli/printers.py

```diff
@@ -220,6 +220,21 @@ class AlgoPrinter(AssetPrinter):
     download_message = 'Download this algorithm\'s code:'
 
 
+class CompositealgoPrinter(AssetPrinter):
```

Comment:
`AlgoPrinter` and `CompositealgoPrinter` have almost the same definition.
What about creating a `BaseAlgoPrinter` and then set `AlgoPrinter` and `CompositealgoPrinter` as child of `BaseAlgoPrinter`

Reply:
Yes, and we can do the same for `TraintuplePrinter` and `CompositeTraintuplePrinter`, with a `BaseTraintuplePrinter`